### PR TITLE
Sample ansible operators update files

### LINKED ISSUE #40: Update sample ansible-operator projects for layout changes
After the controller-runtime refactor we changed the project layout and moved around some manifests around.
Specifically:
- `tmp/build`==>`build`
- `deploy/rbac.yaml`==> `deploy/role.yaml` and `deploy/role_binding.yaml`
- `deploy/cr.yaml` and `deploy/crd.yaml` ==> `deploy/crd/...`

@dymurray @shawn-hurley We should probably update the ansible examples to be in sync with the changes on the master.

### PR DESCRIPTION
**Pull request by johnkim76**
fixes #40

### CODE REVIEW

**Comment by johnkim76** [comment]
fixes #41

**Comment by johnkim76** [comment]
Updated
- ansible/memcached-operator
- ansible/vault-operator
- ansible/bitcoin-sv-operator

Removed
- ansible/galera-operator (still WIP. Available at [water-hole/galera-ansible-operator](https://github.com/water-hole/galera-ansible-operator) 

Unchanged
- ansible/etcd-operator (will not be affected since it uses custom built ansible-operator image)

**Comment by johnkim76** [comment]
> @johnkim76 I think we should go ahead and update the Readme for each operator since some are still just basic role templates. Even if they are just very basic instructions on how to deploy it. I have a template we could probably reuse [here](https://github.com/dymurray/bitcoin-operator/blob/master/README.md) with some minor tweaks.
> 
> I don't mind helping if you want to sync up.

Agreed, I'll update

File: ansible/memcached-operator/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    kubectl create -f deploy/crds/cache_v1alpha1_memcached_cr.yaml
+    ```
+
+## Uninstall
+
+To uninstall the Deployment and the Memcached Ansible Operator, run the following commands
+
+1. Uninstall
+    ```bash
+    kubectl delete -f deploy/crds/bitcoin_v1_bitd_cr.yaml
```
Comment (dymurray) [inline ansible/memcached-operator/README.md]: Just need to replace this line here then should be all set.

Comment (johnkim76) [reply ansible/memcached-operator/README.md]: done. ty!

**Pull request merged**